internal/controller/ec2.aws/securitygroup_controller.go

```diff
@@ -181,7 +188,7 @@ func (r *SecurityGroupReconciliation) retrieveKCPNetworkInfo(ctx context.Context
 		Name:      name,
 		Namespace: namespace,
 	}
-	if err := r.Client.Get(ctx, key, kcp); err != nil {
+	if err := r.Get(ctx, key, kcp); err != nil {
```

Comment:
was this wrong before?

Reply:
It was correct, but we can simplify by omitting it.

Reply:
got it